docs/src/templates/pages/documentation/open-source/creating-repo.hbs

```diff
@@ -0,0 +1,72 @@
+---
+title: Creating a separate GitHub repository
+section-title: Open source
+description:
+docs: true
+
+navgroup: documentation
+navsub: open-source
+navactive: os-creating
+---
+
+<a name="naming-your-module"></a>
+## Naming your module
+
+The general naming convention for a Fozzie package is to use `f-{your-package-name}` e.g. [`f-footer`](https://github.com/justeat/fozzie-components/tree/master/packages/f-footer), [`f-header`](https://github.com/justeat/fozzie-components/tree/master/packages/f-header), [`f-vue-icons`](https://github.com/justeat/fozzie-components/tree/master/packages/f-vue-icons).
+
+
+
+<a name="create-github-repository"></a>
+## Create a new Fozzie GitHub repository
```

Comment:
Maybe need a note in here to say that we prefer the mono-repo approach, with a link to the mono-repo section of the docs?

Reply:
I could change it so that the mono-repo section comes BEFORE the separate repo section. This would make more sense right, as we prefer the mono approach. Thought about this while I was going over it. And incorporate all the things you just mentioned too.